src/main/resources/view/PersonDetailCard.fxml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.FlowPane?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.RowConstraints?>
+<?import javafx.scene.layout.VBox?>
+
+<HBox id="cardPane" fx:id="detailPane" xmlns="http://javafx.com/javafx/19" xmlns:fx="http://javafx.com/fxml/1">
+    <children>
+        <GridPane HBox.hgrow="ALWAYS">
+            <columnConstraints>
+                <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150" />
+            </columnConstraints>
+            <children>
+                <VBox alignment="CENTER_LEFT" minHeight="105" spacing="5.0" GridPane.columnIndex="0">
+                    <padding>
+                        <Insets bottom="5" left="15" right="5" top="5" />
+                    </padding>
+                    <children>
+                        <HBox spacing="5">
+                            <children>
+                                <Label styleClass="cell_big_label" text="Name: ">
+                                    <minWidth>
+                                        <Region fx:constant="USE_PREF_SIZE" />
+                                    </minWidth>
+                                </Label>
+                                <Label fx:id="name" styleClass="cell_big_label" text="\$first" wrapText="true" />
+                            </children>
+                        </HBox>
+                        <FlowPane fx:id="languages" />
```

Comment:
just a slight suggestion for consistency maybe can put 'Languages: ' and 'Tags' in front